app/Mail/BaseMail.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+abstract class BaseMail extends Mailable implements ShouldQueue
```

Comment:
why we can't extends BaseMail from the Helpers package?

Reply:
Because it complitely different. As I remember, I already told you that BaseMail in halpers has an old format (before Laravel 8) and actually doesn't need in me at the Empty Project.